cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -103,6 +103,8 @@ private String getV3RoutingService(String [] uriParts, AllowedMethod requestMeth
     } else if (matches(uriParts, "v3", "security", "authorization")) {
       // Authorization Handler currently runs in App Fabric
       return Constants.Service.APP_FABRIC_HTTP;
+    } else if (matches(uriParts, "v3", "security", "store", "namespaces", null)) {
```

Comment:
Also, add tests for this in ``RouterPathTest``. Its very brittle right now, so we're enforcing unit tests for every condition right now.